scheduler/entrypoint.sh

```
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+if [ -z "$SECRET_KEY" ]; then
+  echo "WARNING: No SECRET_KEY set. Please generate one and export it as SECRET_KEY env variable."
+  export SECRET_KEY=$(tr -dc 'a-z0-9!@#$%^&*(-_=+)' < /dev/urandom | head -c50)
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)